devdoc/hosts.md

```diff
@@ -48,14 +48,6 @@
 - RAM: 16GB
 - Drives: 256GB SSD
 
-### `web01`
-
-- Provider: Hetzner
-- Instance type: [CX31](https://www.hetzner.com/cloud)
-- CPU: 2 vCPUs on Intel Xeon
-- RAM: 8GB
-- Drives: 80GB SSD
-
```

Comment:
Maybe keep this info here, so we have a full list of potential hardware to use.

Reply:
This is a list of machines that we are currently using, I don't understand why we would keep it here?

Reply:
Because not everyone has insights in what hardware we have available. I don't see the advantage in removing it either, but don't consider this as a reason to block this pr.